lib/msf/core/db_manager/http/sinatra_app.rb

```diff
@@ -17,11 +18,20 @@
 require 'msf/core/db_manager/http/servlet/db_export_servlet'
 require 'msf/core/db_manager/http/servlet/vuln_attempt_servlet'
 
+require 'swagger/blocks'
+
 class SinatraApp < Sinatra::Base
+  set :public_folder, File.dirname(__FILE__) + '/public'
+
+  def root_path
```

Comment:
It doesn't appear `root_path` is necessary for this to work as long as your links are based on the root `/` in the erb. Remove uses in `api_docs.erb`.